setup.cfg

```diff
@@ -18,13 +18,8 @@ packages =
 data-files =
     share/ansible-runner/utils = utils/*
 
-[pep8]
-# W503 - Line break occurred before a binary operator
-ignore=W503
-exclude=.tox,venv
-
 [flake8]
 # W503 - Line break occurred before a binary operator
 ignore=W503
 max-line-length=160
-exclude=.tox,venv
+extend-exclude=venv
```

Comment:
Why do we need `venv` here? That isn't part of the repo so seems like we can just remove this line altogether.

Reply:
No idea. Maybe we can just remove this altogether.